code/datums/materials/basemats.dm

```diff
@@ -333,10 +333,15 @@ Unless you know what you're doing, only use the first three numbers. They're in
 
 /datum/material/hot_ice/on_applied(atom/source, amount, material_flags)
 	. = ..()
-	source.AddComponent(/datum/component/hot_ice, "plasma", amount*150, amount*20+300)
+	
+	// Cancel out the material_modifier to get a pure unadulterated material amount
+	var/real_amount = amount
+	if(source.material_modifier)
+		real_amount /= source.material_modifier
+	source.AddComponent(/datum/component/combustible_flooder, "plasma", real_amount*0.075, real_amount*0.01+300)
```

Comment:
```suggestion
	source.AddComponent(/datum/component/combustible_flooder, "plasma", amount*0.075, amount*0.01+300)
```

The entire point of the material modifier is to modify the effects of materials.

Reply:
My original reasoning was for it to preserve the gas amount be it in blocks or sheets. On second thought I suppose that might not be that necessary. I'll come back to this and see which one makes more sense.

Reply:
Status?

Reply:
Will implement Temp's suggestion if this is limited to hot ice.